client/my-sites/domains/paths.js

```diff
@@ -133,10 +133,6 @@ export function domainManagementEmail( siteName, domainName ) {
 	return path;
 }
 
-export function domainManagementEmailForwarding( siteName, domainName ) {
-	return domainManagementEditBase( siteName, domainName, 'email-forwarding' );
-}
-
```

Comment:
This url doesn't seem to be used anywhere so I've removed it. I'm happy to perform a redirect instead if someone feels differently.